pkg/providers/launchtemplate/launchtemplate.go

```diff
@@ -275,6 +294,25 @@ func GetCreateLaunchTemplateInput(options *amifamily.LaunchTemplate, ClusterIPFa
 			},
 		},
 	}
+	// Gate this specifically since the update to CapacityReservationPreference will opt od / spot launches out of open
+	// ODCRs, which is a breaking change from the pre-native ODCR support behavior.
```

Comment:
Good catch -- do we have a test to ensure this behavior is maintained?

Reply:
We will 👍